### DISCUSSION TITLE
code-server 4.17.0 not publish to npm

### DISCUSSION THREAD

**Discussion by 502647092** [coder/code-server#6447] [Q&A] [answered]
code-server 4.17.0 not publish to npm
how to install code-server at alpine

**Comment by vladfrangu** [comment]
Hey @code-asher, sorry for the ping! Could the workflows be re-ran for 4.17.0? Or can a manual run be done? I'm semi affected by this (since brew formula is outdated) 😅

**Comment by code-asher** [comment, marked as answer]
Sorry about this, I fixed the npm publish workflow and 4.17.1 should now be available.

**Reply by vladfrangu** [reply]
Do you happen to know why the brew bump flow failed? 😅 🙏

**Reply by code-asher** [reply]
It fails with `No available formula with the name "code-server".` but I have not done any investigation into it so I have no idea what is going on. :stuck_out_tongue: 

Seems like all we do is use the `Homebrew/actions/setup-homebrew@master` action and then run `brew bump-formula-pr --version="${VERSION}" code-server --no-browse --no-audit --message="PR opened by @${GITHUB_ACTOR}"` but maybe something about the usage changed.
